.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target/
```

Comment:
-1 You should not ignore all target directories, only those with pom.xml
It's perfectly fine to call a java package target or to have a dataset for an example that has a directory called target.